### PR TITLE
feat: password screen abort option

### PR DESCRIPTION
**Pull request by staylorwr**
https://www.loom.com/share/ad55d9c22a044e13b2601e18ee49964b

### CODE REVIEW

File: src/views/Home/index.vue [hunk 1]
```diff
@@ -118,6 +128,23 @@ const CreateWallet = defineComponent({
...
       closeModal () {
         router.push({ name: 'Home', query: { modal: '' } })
+      },
+
+      forgotPassword () {
+        router.push({ name: 'Home', query: { modal: 'forgot-password' } })
+      },
+
+      resetAndCreate () {
+        console.log('create')
```
Comment (caleboller) [inline src/views/Home/index.vue:138]: remove log?

File: src/views/Home/HomeForgotPassword.vue [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    close () {
+      this.mode = 'choice'
+      this.confirmRestore = ''
+      this.confirmCreate = ''
+      console.log('closing')
+      this.$emit('close')
+    },
+
+    submitCreate () {
+      console.log('create')
```
Comment (caleboller) [inline src/views/Home/HomeForgotPassword.vue:152]: remove console.log?

File: src/views/Home/HomeForgotPassword.vue [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      this.mode = 'choice'
+      this.confirmRestore = ''
+      this.confirmCreate = ''
+      this.$emit('resetAndCreate')
+    },
+
+    submitRestore () {
+      console.log('restore')
+      this.mode = 'choice'
```
Comment (caleboller) [inline src/views/Home/HomeForgotPassword.vue:161]: again

File: src/views/Home/HomeForgotPassword.vue [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+      return this.confirmCreate !== 'DELETE WALLET'
+    }
+  },
+
+  methods: {
+    close () {
+      this.mode = 'choice'
+      this.confirmRestore = ''
+      this.confirmCreate = ''
+      console.log('closing')
```
Comment (caleboller) [inline src/views/Home/HomeForgotPassword.vue:147]: again

File: src/actions/electron/data-store.ts [hunk 5]
```diff
@@ -29,3 +29,8 @@ export const saveHardwareAddress = (event: IpcMainInvokeEvent, address: string)
 export const getHardwareAddress = (event: IpcMainInvokeEvent) => {
   return store.get('hardwareAddress')
 }
+
+export const resetStore = (event: IpcMainInvokeEvent) => {
+  console.log('clear the thing')
```
Comment (caleboller) [inline src/actions/electron/data-store.ts:34]: log clean up

File: src/views/Home/HomeForgotPassword.vue [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+        <TransitionChild as="template" enter="ease-out duration-300" enter-from="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95" enter-to="opacity-100 translate-y-0 sm:scale-100" leave="ease-in duration-200" leave-from="opacity-100 translate-y-0 sm:scale-100" leave-to="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95">
+          <div class="inline-block align-bottom bg-white rounded px-4 pt-5 pb-4 text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:align-middle sm:max-w-lg sm:w-full sm:px-4">
+            <div>
+              <div class="mt-3 text-center sm:mt-5">
+                <DialogTitle as="h3" class="text-rRed">
+                  Do you want to delete your current wallet?
+                </DialogTitle>
+                <div class="mt-4 px-8">
+                  <p class="text-rBlack leading-5 text-center">
+                    If you can't remember your password, you'll need to delete this local wallet and restore from your seed phrase or create a new wallet.
```
Comment (caleboller) [inline src/views/Home/HomeForgotPassword.vue:20]: We'll need to do an internationalization clean-up pass after mainnet, I've also been guilt of this.

**Review by caleboller — CHANGES_REQUESTED** [review]
Looks great Scott! There's a few console.logs we should clean up, otherwise this is good to go.

**Comment by staylorwr** [comment]
The logs! 🙈

**Comment by staylorwr** [comment]
Logs and internationalization cleaned up!

**Review by caleboller — APPROVED** [review]

**Pull request merged**